doc/nrf/releases_and_maturity/releases/release-notes-2.6.0.rst

```diff
@@ -0,0 +1,1351 @@
+    * The PSA config is now validated by the file :file:`ncs/nrf/ext/oberon/psa/core/library/check_crypto_config.h`.
+      Users with invalid configurations must update their PSA configuration according to the error messages that the file :file:`check_crypto_config.h` provides.
+
+Other libraries
+---------------
+
+* :ref:`lib_adp536x` library:
+
+  * Fixed the issue where the adp536x driver was included in the immutable bootloader on Thingy:91 when :kconfig:option:`CONFIG_SECURE_BOOT` was enabled.
+
+* :ref:`mod_memfault` library:
+
+  * Added more default LTE metrics, such as band, operator, RSRP, and kilobytes sent and received.
+  * Updated the default metric names to follow the standard |NCS| variable name convention.
+
+* :ref:`lib_date_time` library:
+
+  * Added the :c:func:`date_time_now_local` function to the API.
+
+Common Application Framework (CAF)
+----------------------------------
+
+* :ref:`caf_ble_state`:
+
+  * Updated the dependencies of the :kconfig:option:`CONFIG_CAF_BLE_USE_LLPM` Kconfig option.
+    The option can be enabled even when the Bluetooth controller is not enabled as part of the application that uses :ref:`caf_ble_state`.
+  * Added :c:member:`ble_peer_event.reason` to inform about reason code related to state of the Bluetooth LE peer.
+    The field is used to propagate information about error code related to a connection establishment failure and disconnection reason.
+
+* :ref:`caf_ble_adv`:
+
+  * Added the :kconfig:option:`CONFIG_CAF_BLE_ADV_POWER_DOWN_ON_DISCONNECTION_REASON_0X15` Kconfig option.
+    You can use this option to force system power down when a bonded peer disconnects with reason ``0x15`` (Remote Device Terminated due to Power Off).
+
+sdk-nrfxlib
+-----------
+
+See the changelog for each library in the :doc:`nrfxlib documentation <nrfxlib:README>` for additional information.
+
+Scripts
+=======
+
+This section provides detailed lists of changes by :ref:`script <scripts>`.
+
+* :ref:`nrf_desktop_config_channel_script`:
+
+  * Separated functions that are specific to handling the :file:`dfu_application.zip` file format.
+    The ZIP format is used for update images in the nRF Connect SDK.
+    The change simplifies integrating new update image file formats.
+
+* Added the :file:`cert_tool.py` script.
+  This is a script to generate EC private keys, create CSRs, create root CA and subordinate CA certificates and sign CSRs.
+
+MCUboot
+=======
+
+The MCUboot fork in |NCS| (``sdk-mcuboot``) contains all commits from the upstream MCUboot repository up to and including ``11ecbf639d826c084973beed709a63d51d9b684e``, with some |NCS| specific additions.
+
+The code for integrating MCUboot into |NCS| is located in the :file:`ncs/nrf/modules/mcuboot` folder.
+
+The following list summarizes both the main changes inherited from upstream MCUboot and the main changes applied to the |NCS| specific additions:
+
```

Comment:
@nvlsianpu could you check if there is anything to be added here?

Reply:
- Added CONFIG_XIP_SPLIT_IMAGE optionn which enables build systemsupport for relopcate part of the application to external memory using hardware QSPI XIP feature and MCUboot 3rd image on nRF5340 SoC.
- Added mcuboot's procedure which performs clean-up off content of all the secondary slots which contains valid header but couldn't be assigned to any of supported primary images. This behaviour is desired when configuration allows to use one secondary slot for collecting image for multiple primary slots. Ref.: config CONFIG_MCUBOOT_CLEANUP_UNUSABLE_SECONDARY, NCSIDB-1151

Reply:
Added